.goreleaser.yml

```diff
@@ -14,6 +14,7 @@ builds:
       - darwin
     goarch:
       - amd64
+      - arm
```

Comment:
is this one meant to be darwin? all the other additions are linux

Reply:
oooooops, you are correct sir.

Reply:
fixed.